src/Framework/TestCase.php

```diff
@@ -187,7 +187,7 @@
     /**
      * The code of the expected Exception.
      *
-     * @var int|string
+     * @var null|int|string
```

Comment:
it's not initialized with any value, if one would not call `expectException`, then this would remain null.